src/lrud.test.js

```diff
@@ -459,6 +459,23 @@ describe('lrud', () => {
     })
   })
 
+  describe('setActiveChild', () => {
+    it('does not set the parent\'s parent\'s activeChild property if `shallow` flag provided', () => {
+      const navigation = new Lrud()
+      navigation
+        .registerNode('root')
+        .registerNode('a', { parent: 'root' })
+        .registerNode('b', { parent: 'root' })
+        .registerNode('a0', { isFocusable: true, parent: 'a' })
+        .registerNode('b0', { isFocusable: true, parent: 'b' })
+
+      navigation.assignFocus('b0')
+      navigation.setActiveChild('a', 'a0', { shallow: true })
```

Comment:
This `shallow` flag isn't necessary is it? The behaviour has been changed outright.

Reply:
Good catch! I had when I implemented it the other way.